api/models/fixtures/agent_label.json

```diff
@@ -75,6 +75,42 @@
         },
         "Yes"
       ]
+    },
+    "RegionHasCustomTopicUpdate": {
```

Comment:
@ddollar I don't see how to avoid a conditional of some kind. We can't just set the runtime to whatever is in the RegionConfig map because it would override the Parameter setting for newer Racks.

We still need to be able to do an If like this somehow:

```
          "Fn::If": [
             "RegionHasCustomTopicUpdate",
             "nodejs4.3",
             {
               "Ref": "CustomTopicRuntime"
             }
           ]
```